adapter/src/dbt/adapters/fal/connections.py

```diff
@@ -23,7 +44,9 @@ def cancel(self, connection):
 class FalCredentials(Credentials):
     default_environment: str = "local"
 
-    # NOTE: So we can not set them in profiles.yml
+    teleport: Optional[TeleportCredentials] = None
```

Comment:
ATL: does DBT allow nested credentials?

Reply:
Well, it is not _really_ a credentials object in the same sense that the FalCredentials are. It's just the way to access/config Teleport; but in this case dbt just sees it as an object to parse.